app/views/layouts/provider.html.slim

```diff
@@ -6,6 +6,9 @@ html[lang="en" class="pf-m-redhat-font"]
       = content_for?(:title) ? yield(:title) : default_title
       |  | Red Hat 3scale API Management
     = csrf_meta_tag
+    // Is this the entrypoint of the app?
```

Comment:
I think it's not the only layout that is used in the app... probably need to add this in others as well.

Reply:
I think I updated it in all layouts